### PR TITLE
feat(entrypoints): add entrypoints from panel and Explore

### PR DESCRIPTION
**Pull request by svennergr**
More of a hack day project. This PR adds entrypoints from panels and Explore:

https://github.com/user-attachments/assets/e6a4f207-faed-419f-9494-e7db4c8d69eb

### CODE REVIEW

File: src/services/extensions/links.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+export const linkConfigs: PluginExtensionLinkConfig[] = [
+  {
+    extensionPointId: PluginExtensionPoints.DashboardPanelMenu,
+    title,
+    description,
+    icon,
+    path: createAppUrl(),
+    configure: (context?: PluginExtensionPanelContext) => {
+      if (!context) {
```
Comment (matyax) [inline src/services/extensions/links.ts:20]: Lines 20:40 and 66:86 seem to be doing the same thing. Maybe extract to a common place?

File: src/services/routing.ts [hunk 2]
```diff
@@ -138,3 +138,33 @@ export function buildServicesRoute(extraQueryParams?: UrlQueryMap): UrlQueryMap
     ...extraQueryParams,
   };
 }
+
+export function createAppUrl(path = '/explore', urlParams?: URLSearchParams): string {
+  return `/a/${pluginJson.id}${path}${urlParams ? `?${urlParams.toString()}` : ''}`;
+}
+
+export enum UrlParameterType {
+  DatasourceId = 'var-ds',
```
Comment (matyax) [inline src/services/routing.ts:147]: Can be taken from variables.ts, right?

Comment (matyax) [reply src/services/routing.ts:147]:
```suggestion
  DatasourceId = `var-${VAR_NAME}`,
```

Not sure if this is possible with TS.

**Review by matyax — COMMENTED** [review]
Looks great but couldn't get it to work locally.

File: src/services/routing.ts (hunk 2 above)
Comment (svennergr) [reply src/services/routing.ts]: Yea, it's not, but I switched to using a `const`. Good recommendation.

File: src/services/extensions/links.ts (hunk 1 above)
Comment (svennergr) [reply src/services/extensions/links.ts]: Sounds good!

**Comment by svennergr** [comment]
@matyax in order to test, since this needs preloading of the plugin, make sure to build the plugin first and then start Grafana. Or restart Grafana after the plugin was built.

**Review by matyax — APPROVED** [review]
:rocket:

**Pull request merged**